packages/cubejs-playground/src/components/PlaygroundQueryBuilder/QueryBuilderContainer.tsx

```diff
@@ -67,20 +67,26 @@ export function QueryBuilderContainer({
 
   return (
     <CubeProvider cubejsApi={cubejsApi}>
-      <RollupDesignerContext apiUrl={apiUrl!}>
-        <ChartRendererStateProvider>
-          <StyledCard bordered={false}>
-            <QueryTabsRenderer
-              apiUrl={apiUrl!}
-              token={currentToken!}
-              dashboardSource={props.dashboardSource}
-              securityContextToken={securityContextToken}
-              onTabChange={props.onTabChange}
-              onSecurityContextModalOpen={() => setIsModalOpen(true)}
-            />
-          </StyledCard>
-        </ChartRendererStateProvider>
-      </RollupDesignerContext>
+      <CubeContext.Consumer>
+        {({ cubejsApi }) =>
```

Comment:
Here I am just waiting `cubejsApi` to be ready, since I am using `useCubeQuery` in children components and the initial render => `cubejsApi = null`